docs/user/pytorch.md

```diff
@@ -0,0 +1,77 @@
+# DAOS pytorch interface
```

Comment:
You need to add an entry for this file to https://github.com/daos-stack/daos/blob/master/mkdocs.yml